qldbdriver/qldbdriver.go

```diff
@@ -71,6 +71,11 @@ func New(ledgerName string, qldbSession *qldbsession.QLDBSession, fns ...func(*D
 		return nil, &qldbDriverError{"MaxConcurrentTransactions must be 1 or greater."}
 	}
 
+	// Ensure logger's internal verbosity is consistent with the option's verbosity for when defaultLogger is used.
```

Comment:
This is not how the current logging works. This block of code is not needed.

Reply:
Addressed.